src/hevm/src/EVM.hs

```diff
@@ -1895,19 +2023,21 @@ mkCodeOps bytes = RegularVector.fromList . toList $ go 0 bytes
 -- Gas cost function for CALL, transliterated from the Yellow Paper.
 costOfCall
   :: FeeSchedule Word
-  -> Maybe a -> Word -> Word -> Word
+  -> Bool -> Word -> Word -> Word
   -> (Word, Word)
-costOfCall (FeeSchedule {..}) recipient xValue availableGas xGas =
+costOfCall (FeeSchedule {..}) recipientExists xValue availableGas xGas =
```

Comment:
What was wrong with Maybe?

Reply:
`costOfCall` used to take a `Maybe Contract` but only checked `isNothing`, I think this was sloppy (and sometimes burdensome to actually produce a term of that type) so I changed it to take a `Bool`.